corehq/apps/translations/app_translations.py

```diff
@@ -209,25 +211,47 @@ def validate_bulk_app_translation_upload(app, workbook, email, file_obj):
     msgs = UploadedTranslationsValidator(app, workbook).compare()
     checker_messages, result_wb = run_translation_checker(file_obj)
     if msgs:
-        _email_app_translations_discrepancies(msgs, email, app.name)
+        _email_app_translations_discrepancies(msgs, checker_messages, email, app.name, result_wb)
         return [(messages.error, _("Issues found. You should receive an email shortly."))]
     else:
         return [(messages.success, _("No issues found."))]
 
 
-def _email_app_translations_discrepancies(msgs, email, app_name):
-    html_content = ghdiff.default_css
-    for sheet_name, msg in msgs.items():
-        html_content += "<strong>{}</strong>".format(sheet_name) + msg
+def _email_app_translations_discrepancies(msgs, text_messages, email, app_name, result_wb):
```

Comment:
nit: you could use the same variable name here and above (text_messages vs checker_messages) for simplicity